openmdao/api.py

```diff
@@ -40,10 +39,8 @@
 from openmdao.solvers.run_once import RunOnce
 from openmdao.solvers.scipy_gmres import ScipyGMRES
 from openmdao.solvers.solver_base import LinearSolver, NonLinearSolver
-try:
+if under_mpirun():
     from openmdao.solvers.petsc_ksp import PetscKSP
```

Comment:
Ok, it seems that this dirty hack adds the needed environment variables:

    eval "export `mpirun env | grep OMPI_MCA_orte_precondition_transports`"

http://comments.gmane.org/gmane.comp.clustering.open-mpi.user/14871

So if this is simply just our cluster acting up, please ignore the pull request.